src/routes.py

```diff
@@ -1,17 +1,19 @@
 import logging
 from flask import request, jsonify, render_template
 from . import flask_app
-from .token_utilities import process_token, get_cache, opaque_token_count
+from .token_utilities import process_token, get_cache
+
+OPAQUE_TOKEN_COUNT_KEY = "opaque_token_count"
```

Comment:
Store this in a config file, don't leave it lying around in random files.

For example, __init__.py or config.py